packages/map/enable.js

```diff
@@ -0,0 +1,3 @@
+window.Vaadin = window.Vaadin || {};
+window.Vaadin.experimental = window.Vaadin.experimental || {};
+window.Vaadin.experimental.enableMapComponent = true;
```

Comment:
Sort of adapted from the properties for Flow feature flags: https://github.com/vaadin/flow/blob/master/flow-server/src/main/java/com/vaadin/experimental/FeatureFlags.java#L271